Packs/ExodusIntelligence/.secrets-ignore

```diff
@@ -0,0 +1,2 @@
+hxxps://vpx.exodusintel.com
```

Comment:
I now understand that the secret is for the actual URL you are providing to customer, I am sorry for misunderstanding. 

```suggestion
https://vpx.exodusintel.com
```

Reply:
done